workflow/python/covid19_scrapers/states/kansas.py

```diff
@@ -28,8 +28,13 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
     def get_date(self, soup):
-        last_updated_text = soup.find(text='Last updated:')
-        date_str = last_updated_text.find_next('span').text
+        # the date string is represented by 3 different <span> elements
+        current_element = soup.find(text='Last updated')
+        date_str = ''
+        for _ in range(3):
```

Comment:
Just checked out the HTML -- `01` in one span and `/15/2020` in the one after next... wow.
I think it might be a little more robust to get the `Last updated` span's parent's `get_text()`, and use the first match for the pattern as the date, rather than extracting a fixed number of elements' text.

Reply:
(If you don't like this, just resolve the conversation.)

Reply:
Ooh, yeah that sounds a lot cleaner to me, will do!